types/is-boolean-object/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for is-boolean-object 1.1
+// Project: https://github.com/inspect-js/is-boolean-object
+// Definitions by: aaronleopold <https://github.com/aaronleopold>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isBoolean(value: any): boolean;
```

Comment:
```suggestion
declare function isBoolean(value: any): value is boolean;
```
you can mark this as type predicate, to better signal what's behind:
https://dev.to/daveturissini/aha-understanding-typescript-s-type-predicates-40ha#:~:text=Type%20predicates%20are%20a%20special%20return%20type%20that,Type%20predicates%20are%20expressed%20as%20argumentName%20is%20Type.